src/TimelineEvent/index.scss

```diff
@@ -0,0 +1,73 @@
+@keyframes flash {
+  from {
+    background-color: var(--theme-secondary);
+  }
+  to {
+    background-color: var(--color-lightGrey);
+  }
+}
+
+.nds-timeline-event {
+  padding-bottom: var(--space-l);
+  display: flex;
+  align-items: stretch;
+  gap: var(--space-xs);
+}
+
+.nds-timeline-track {
+  position: relative;
+  min-width: 56px;
+  flex-grow: 0;
+}
+
+.nds-timeline-content {
+  flex-grow: 1;
+}
+
+.nds-timeline-line {
+  position: absolute;
+  z-index: 0;
+  left: 0;
+  right: 0;
+  // height of event + gap
+  height: calc(100% + var(--space-l));
```

Comment:
This matches the height of the SVG to the height of the root element in `TimelineEvent`, plus the height of a node, creating the overflap